tests/minimax.wcon

```diff
@@ -27,7 +27,10 @@
 		"ox":"mm",
 		"speed":"mm/s",
 		"curvature":"1/mm",
-		"width":"mm"
+		"width":"mm",
+		"humidity":"%",
+		"temperature":"C",
+        "age":"h"
```

Comment:
Agree - I think this was a case of my editing this file with Notepad++ instead of my usual Spyder or vim.  Thanks for catching this; I'll fix it.